bitnami/airflow-scheduler/README.md

```diff
@@ -61,14 +61,14 @@ If you want to run the application manually instead of using `docker-compose`, t
 1. Create a network
 
   ```console
-  $ docker network create airflow-tier
+  docker network create airflow-tier
   ```
 
-2. Create a volume for PostgreSQL persistence and create a PostgreSQL container
+1. Create a volume for PostgreSQL persistence and create a PostgreSQL container
```

Comment:
The ordered list is not auto incrementing the index

Reply:
The default style is `one` according to https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix

Reply:
GitHub markdown may not be following that convention, right? See the rendered section: https://github.com/bitnami/containers/blob/e7ea14f56da588fefb95a24693ee396580f7347e/bitnami/airflow-scheduler/README.md#using-the-docker-command-line all the ordered items show the `1.` prefix.

Is it possible to configure the markdown linter to comply with the style required by GitHub?

Reply:
I will replace it by ordered lists in the following commit